src/core/ack_tracker.h

```diff
@@ -100,7 +100,7 @@ QuicAckTrackerAckPacket(
     _Inout_ QUIC_ACK_TRACKER* Tracker,
     _In_ uint64_t PacketNumber,
     _In_ uint64_t RecvTimeUs,
-    _In_ QUIC_ECN_TYPE ECN,
+    _In_ CXPLAT_ECN_TYPE ECN,
```

Comment:
Is ECN a quic specific thing, or is that applicable to tcp/udp?

Reply:
It is **IP** specific, so not QUIC specific.